src/gui/tray/usermodel.cpp

```diff
@@ -506,6 +506,8 @@ void User::processCompletedSyncItem(const Folder *folder, const SyncFileItemPtr
     activity._folder = folder->alias();
     activity._fileAction = "";
 
+    const auto fileName = QFileInfo(item->_originalFile).fileName();
```

Comment:
should you not also check that `QFileInfo(item->_originalFile).exists()` is true ?

Reply:
No, since both _originalFile and _file are not actual file locations on disk and are relative to the syncfolder

Checks that the file still exists are done in the ActivityListModel where we actually put together the real file location

Reply:
so you lost me here
if `item->_originalFile` is not an existing file what is returned by `QFileInfo(item->_originalFile).fileName()` and why do we need it ?

Reply:
We use the fileName here for the activity's `message`, originally we used `item->_originalFile` for the `message` which provided a file path relative to the sync folder

Reply:
I would really have expected `fileName()` to only work with existing files but apparently not
I tested
```
qDebug() << QFileInfo("/blih/bluh").fileName();
```

this file is nowhere on my system but this call is working